docs/fabtoken.md

```diff
@@ -1,3 +1,52 @@
 # FabToken
 
-To Be Continued...
\ No newline at end of file
+The `FabToken` driver is a simple implementation of the Driver API that does not support privacy.
+Therefore, the ledger reveals to any third party who did what. 
+
+## Public Params Manager
+
+`FabToken` understands the following public parameters:
+
+```go
+type PublicParams struct {
+	// This indicates that this is the public parameters for fabtoken driver
+	Label string
+	// The precision of token quantities
+	QuantityPrecision uint64
+	// This is set when audit is enabled
+	Auditor []byte
+	// This encodes the list of authorized auditors
+	Issuers [][]byte
+}
+```
+
+The `Label` field must be set to `"fabtoken"`.
+`FabToken` supports multiple issuers and a single auditor.
+
+## IdentityProvider
+
+In `FabToken`, the only long-term identities supported are `X509-based Fabric MSP identities`.
+Such an identity contains an X509 certificate and reveal in the clear the Enrollment ID of the certificate's owner.
```

Comment:
Yes, it is a Fabric specific term. For example, Alice's certificate carries her Enrollment ID. It can be anything that uniquely identify Alice in a given context.